models/platform-limits/src/limit-with-current/presets/with-exceeded.ts

```diff
@@ -0,0 +1,5 @@
+import LimitWithCurrent from '../builder';
+
+const withExceeded = () => LimitWithCurrent().limit(10).current(10);
```

Comment:
Shouldn't `current` be a greater number than `limit`?

Reply:
Based on this [util file](https://github.com/commercetools/merchant-center-frontend/blob/main/packages-shared/experimental-components/src/features/platform-limits/shared/utils/utils.js), if the both value are same, then it already exceeded